src/components/panel/panel.js

```diff
@@ -1153,20 +1156,24 @@ MdPanelRef.prototype.open = function() {
 
 /**
  * Closes the panel.
+ * @param {string} closeEvent The event type that triggered the close.
```

Comment:
@ErinCoughlan Good catch! I was thinking that it was going to be the event that the occurred to get there, not a string representation.